vllm/multimodal/image.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict, Type
+
+import numpy as np
+import torch
+from PIL import Image
+
+from vllm.config import ModelConfig, VisionLanguageConfig
+from vllm.logger import init_logger
+from vllm.sequence import SequenceData
+from vllm.transformers_utils.image_processor import cached_get_image_processor
+
+from .base import MultiModalData, MultiModalPlugin
+
+logger = init_logger(__name__)
+
+
+def _get_seq_data(seq_len: int, vlm_config: VisionLanguageConfig):
```

Comment:
Is this only for dummy data generation? If so let's rename it
```suggestion
def _get_dummy_seq_data(seq_len: int, vlm_config: VisionLanguageConfig):
```